src/api/contact-center/queues/queues.js

```diff
@@ -76,28 +70,29 @@ export const getQueue = async (id) => {
             name: strategiesList[response.strategy],
             value: response.strategy,
         };
-        return {...defaultObject, ...response};
+        return {...defaultItemObject, ...response};
     } catch (err) {
         throw err;
     }
 };
 
+export const getQueuesList = async (page = 0, size = 10, search) => {
+    return await listGetter.getList({page, size, search});
+};
+
+export const getQueue = async (id) => {
+    return await itemGetter.getItem(id);
+};
+
 export const addQueue = async (item) => {
     let itemCopy = deepCopy(item);
     itemCopy.domainId = store.state.userinfo.domainId || undefined;
```

Comment:
remove || undefined + ctrl+shift+f to find and remove all "|| undefined" occurencies

Reply:
ctrl+shift+r to replace "|| undefined" to ""